core/shared/platform/nuttx/platform_internal.h

```diff
@@ -68,6 +68,13 @@ typedef pthread_t korp_thread;
 #define O_NOFOLLOW 0
 #endif
 
+#undef CONFIG_HAS_ISATTY
```

Comment:
Is it enough to override globally ?

Reply:
CONFIG_HAS_ISATTY is defind in ssp_config.h to indicate that isatty is enabled, and only refered once in posix.c. ssp_config.h including is always ahead of bh_platform.h therefore all of them would be overridden